docker/dev/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASEIMAGE=nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04
+ARG BASEIMAGE=nvidia/cuda:11.0-cudnn8-devel-ubuntu18.04
```

Comment:
cu100 Docker removed? any comments @Yancey1989 ？

Reply:
All the CI jobs specified this `BASEIMAGE argument` as `--build-arg`, just like https://github.com/alibaba/BladeDISC/blob/main/.github/workflows/main.yml#L126 .  @fortianyou maybe we don't need to modify the default value if you want to update the CI job behaviors.

Reply:
Reverted